apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/entity/WaTemplate.java

```diff
@@ -46,7 +48,7 @@ public class WaTemplate {
   private static final long TAB = 1010l;
   private static final long SECTION = 1015l;
   private static final long SUB_SECTION = 1016l;
-  private static final long QUESTION = 1007l;
+  private static final long QUESTION = 1008l;
```

Comment:
Potential Issue (needs checked): I think this might be wrong. Several of the page components have an id that wont change (such as TAB, SECTION, SUB_SECTION). A question could be one of several component 
|id | component type|
|---|----|
|1006 | radio button|
|1007 | Single select drop down|
|1008 | user entered text|
|1009 | multiline text|

This list is only partial. It might be a good idea to find the group / ungroup logic within Classic and try to verify how its working.  It might be the case that the check for the component type of question is either not needed, or can be inverted to check that the component is not one of a set of values.